src/main/java/se/sundsvall/billingpreprocessor/Constants.java

```diff
@@ -28,4 +26,7 @@ private Constants() {}
 	public static final String ERROR_SUBACCOUNT_NOT_PRESENT = "Sub account is not present";
 	public static final String ERROR_TOTAL_AMOUNT_NOT_PRESENT = "Total amount is not present";
 	public static final String ERROR_VAT_CODE_NOT_PRESENT = "Vat code is not present";
+
+	public static final String EXTERNAL_INVOICE_TYPE = "Extern debitering";
```

Comment:
Might want to stick to english.

Reply:
If it was up to me I would, but this is the string that defined by Raindance to be present in the file created for Raindance.